docs/sources/tempo/traceql/_index.md

```diff
@@ -22,37 +22,41 @@ Read the blog post, [Get to know TraceQL](/blog/2023/02/07/get-to-know-traceql-a
 
 {{< vimeo 796408188 >}}
 
-For information on where the language is headed, see [future work]({{< relref "./architecture#future-work" >}}).
 The TraceQL language uses similar syntax and semantics as [PromQL](/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/) and [LogQL](/docs/loki/latest/logql/), where possible.
 
-Check the [release notes]({{< relref "../release-notes" >}}) for the latest updates to TraceQL.
+Check the [Tempo release notes](https://grafana.com/docs/tempo/<TEMPO_VERSION>/release-notes/) for the latest updates to TraceQL.
 
 ## Requirements
 
-TraceQL requires Tempo’s Parquet columnar format, which is enabled by default. For information on Parquet, refer to the [Apache Parquet backend]({{< relref "..//configuration/parquet" >}}) Tempo documentation.
+TraceQL requires the Parquet columnar format, which is enabled by default.
+For information on Parquet, refer to the [Apache Parquet backend](http://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/parquet) documentation.
 
-## TraceQL queries in Grafana Explore
+## Query using TraceQL
 
-You can use the TraceQL query editor and query builder in the Tempo data source to build queries and drill-down into result sets.
-The editor and builder are available in Grafana’s Explore interface.
-For more information, refer to [Write TraceQL queries in Grafana]({{< relref "./query-editor" >}}).
+You can use TraceQL query editor and query builder in the Tempo data source to build queries and drill-down into result sets.
+The editor and builder are available in the [Tempo data source](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/) for Grafana Explore.
 
 <p align="center"><img src="assets/query-editor-http-method.png" alt="Query editor showing request for http.method" /></p>
 
+In addition, you can use Explore Traces to investigate your tracing data without writing TraceQL queries.
+For more information, refer to the [Explore Traces](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/traces/) documentation.
```

Comment:
Does <GRAFANA_VERSION> need to be replaced?

Reply:
Nah, it gets replaced during link rendering by the value in https://github.com/grafana/tempo/blob/main/docs/sources/tempo/_index.md?plain=1#L6-L7